docs/fundamentals/single-project.md

```diff
@@ -1,35 +1,195 @@
+
+```xml
+<MauiAsset Include="Resources\Assets\index.html" />
+```
+
+Raw assets can then be consumed by controls, as required:
+
+```xaml
+<WebView Source="index.html" />
+```
+
+At build time, raw assets are copied to your app package.
+
+### CSS files
+
+Cascading Style Sheet (CSS) files can be added to your app project by dragging them into the _Resources_ folder of your project, and setting their build action to `MauiCss` in the **Properties** window. This creates a corresponding entry per CSS file in your project file:
+
+```xml
+<MauiCss Include="Resources\styles.css" />
+```
+
+CSS files must be loaded by the `StyleSheet` class before being added to a `ResourceDictionary`:
+
+```xaml
+<Application ...>
+    <Application.Resources>
+        <StyleSheet Source="styles.css" />
+    </Application.Resources>
+</Application>
+```
+
+<!-- For more information, see [](). -->
+
+## Platform-specific code
+
+A .NET MAUI app project contains a _Platforms_ folder, with each child folder representing a platform that .NET MAUI can target:
+
+:::image type="content" source="media/single-project/platform-folders.png" alt-text="Platform folders screenshot.":::
+
+The folders for each platform contain platform-specific resources, and code and that starts the app on each platform:
+
+:::image type="content" source="media/single-project/platform-code.png" alt-text="Platform-specific code screenshot.":::
+
+At build time, the build system only includes the code from each folder when building for that specific platform. For example, when you build for Android the files in the _Platforms\Android_ folder will be built into the app package, but the files in the other _Platforms_ folders won't be. This approach uses multi-targeting to target multiple platforms from a single project. Multi-targeting can be combined with partial classes and partial methods to invoke native platform functionality from cross-platform code. For more information, see [Invoke platform code](~/platform-integration/invoke-platform-code.md).
+
+In addition to this default multi-targeting approach, .NET MAUI apps can also be multi-targeted based on your own filename and folder criteria. This enables you to structure your .NET MAUI app project so that you don't have to place your platform code into sub-folders of the _Platforms_ folder. For more information, see [Configure multi-targeting](~/platform-integration/configure-multi-targeting.md).
+
+Multi-targeting can also be combined with conditional compilation so that code is targeted to specific platforms:
+
+```csharp
+#if __ANDROID__
+                  handler.NativeView.SetBackgroundColor(Colors.Red.ToNative());
+#elif __IOS__
```

Comment:
You should drop the `__` on these and just use `ANDROID` and `IOS`. These are the new defines provided by the .NET SDK, spec here:

https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#vary-implementation

`__ANDROID__` and the other one still *work*, we kept them for compatibility with Xamarin apps. But I don't think any documentation should show these.